### PR TITLE
Lberner ado scrub terms in security docs

### PR DESCRIPTION
**Pull request by LouisBerner**

### CODE REVIEW

**Comment by Jak-MS** [comment]
@alkohli 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by LouisBerner** [comment]
Closing as not needed.

File: azure-stack/hci/manage/manage-bitlocker.md [hunk 1]
```diff
@@ -1,42 +1,55 @@
 ---
-title: Manage BitLocker encryption on Azure Stack HCI, version 23H2 (preview)
-description: Learn how to manage BitLocker encryption on your Azure Stack HCI, version 23H2 system (preview).
+title: Manage volume encryption with BitLocker on Azure Stack HCI, version 23H2 (preview)
```
Comment (carlosmayol) [inline azure-stack/hci/manage/manage-bitlocker.md:2]: please keep the old title, our code manages more than just VOLUMES.

File: azure-stack/hci/manage/manage-bitlocker.md [hunk 2]
```diff
@@ -73,9 +86,9 @@ Follow these steps to disable BitLocker encryption:
     Disable-ASBitLocker
     ```
 
-## Get BitLocker recovery keys
+## Get recovery keys for volume encryption with BitLocker
```
Comment (carlosmayol) [inline azure-stack/hci/manage/manage-bitlocker.md:89]: Same as above, please leave the old and dont rename to volume encryption with BL...

**Pull request closed without merging**